restapi/src/main/java/io/stargate/web/docsapi/service/DocumentService.java

```diff
@@ -283,7 +283,7 @@ private boolean isEmptyArray(Object v) {
       String leaf = null;
       for (int i = 0; i < fieldNames.length; i++) {
         String fieldName = fieldNames[i];
-        boolean isArrayElement = fieldName.startsWith("[");
+        boolean isArrayElement = fieldName.startsWith("[") && fieldName.endsWith("]");
```

Comment:
This fixes just a bunch of Bad Request-type errors due to randomized input from YCSB (they actually end up slowing down throughput when they fail :( )